pkgs/by-name/me/meritous/package.nix

```diff
@@ -1,20 +1,21 @@
 { lib, stdenv, fetchFromGitLab, SDL, SDL_image, SDL_mixer, zlib }:
 
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "meritous";
-  version = "1.4";
+  version = "1.5";
 
   src = fetchFromGitLab {
     owner = "meritous";
     repo = "meritous";
-    rev = "314af46d84d2746eec4c30a0f63cbc2e651d5303";
-    sha256 = "1hrwm65isg5nwzydyd8gvgl3p36sbj09rsn228sppr8g5p9sm10x";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-6KK2anjX+fPsYf4HSOHQ0EQBINqZiVbxo1RmBR6pslg=";
   };
 
   prePatch = ''
     substituteInPlace Makefile \
-      --replace "CPPFLAGS +=" "CPPFLAGS += -DSAVES_IN_HOME -DDATADIR=\\\"$out/share/meritous\\\"" \
```

Comment:
Note for reviewers: `SAVES_IN_HOME` environment variable has been removed upstream